src/OmniSharp.Roslyn.CSharp/Services/Buffer/UpdateBufferService.cs

```diff
@@ -20,7 +20,7 @@ public UpdateBufferService(OmniSharpWorkspace workspace)
 
         public async Task<object> Handle(UpdateBufferRequest request)
         {
-            await _workspace.BufferManager.UpdateBuffer(request);
+            await _workspace.BufferManager.UpdateBufferAsync(request);
```

Comment:
Note that I'm not make the Handle method async. I think that's overkill. However, I ran into an issue where I'd failed to await a Task returned from a method because it does not state it was Async, which caused me an hour of debugging to find the source of a bizarre re-entrancy problem. When I realized my fail, I went ahead and renamed. :smile: